source/agora/utils/Test.d

```diff
@@ -589,3 +591,174 @@ unittest
         assert(!verify(pz.V, ssz, "WK.Keys.z".representation));
     }
 }
+
+/*******************************************************************************
+
+    Generate a new transaction that evenly splits the input accross parties
+
+*******************************************************************************/
+
+public Transaction split (TxType type = TxType.Payment)
```

Comment:
Note to reviewers: This does not need to be a template parameter, but it simplifies the API IMHO.